Automattic/wp-calypso
# Introduce a Calypso environment for hosts.automattic.com

## Proposed Changes

* Adds new config files for the following environments: `a8c-for-hosts-development`, `a8c-for-hosts-stage`, `a8c-for-hosts-production`.
* Adds new section and and router which renders "Hello world" in the appropriate place.
* Mimics A4A environment badges on the new environments.
* Adds scripts to do the docker build, start the server, etc.
* Adds `hosts.automattic.com` and `hosts.localhost` as allowed origins.

## Testing Instructions

1. Apply D156071-code and sandbox the API.
2. Switch to this branch and start the server with `start-a8c-for-hosts`.
3. Use the live links to verify that all the other Calypso branches start and load correctly (A4A, WPCOM, and Jetpack Cloud).
4. Visit `http://hosts.localhost:3000` and verify you see `Hello, world`.

## Pre-merge Checklist
